### PR TITLE
Add sidebar accessibility

### PR DESCRIPTION
**Pull request by BohdanBilonoh**
- [x] Updat SDK to 2.20
- [x] Add sample for sidebar accessibility API

### CODE REVIEW

File: NewTutorials/Podfile [hunk 1]
```diff
@@ -2,10 +2,10 @@ source "https://github.com/vidible/OneMobileSDK-releases.git"
 
 target 'Tutorial' do
 	platform :ios, 8.0
-  	pod 'OneMobileSDK-Xcode9.2'
+  	pod 'OneMobileSDK-Xcode9.2', '~> 2.20.2141'
```
Comment (AndriiMoskvin) [inline NewTutorials/Podfile]: Why we need to specify this version? `pod 'OneMobileSDK-Xcode9.2'` will do the same.

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

File: NewTutorials/Tutorial/Players.swift [hunk 2]
```diff
@@ -34,7 +34,6 @@ func videoWithoutAutoplay() -> Future<Result<Player>> {
 func liveVideo() -> Future<Result<Player>> {
     return OneSDK.Provider.default.getSDK()
         .then { $0.getPlayer(videoID: "5a600ab092fdde68e1f7a606") }
-//        .then { $0.getPlayer(videoID: "59833447b90afb42310c19da") }
```
Comment (AndriiMoskvin) [inline NewTutorials/Tutorial/Players.swift:37]: Why this is greyed out?

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

File: NewTutorials/Tutorial/Players.swift (hunk 2 above)
Comment (BohdanBilonoh) [reply NewTutorials/Tutorial/Players.swift:37]: Because it's old and bad live video

Comment (AndriiMoskvin) [reply NewTutorials/Tutorial/Players.swift:37]: Ok, got it.

**Review by AndriiMoskvin — APPROVED** [review]

**Pull request merged**